index.php

```diff
@@ -0,0 +1,64 @@
+<?php
+require_once __DIR__ . '/vendor/autoload.php';
+
+$db = new TopCat\Utilities\DB();
+$dbConnection = $db->dbConnect();
+
+// Prepare target strings
+$dropdownBreeds = '';
+$catshtml = '';
+
+// Populate the dropdown with the list of breeds from the DB
+$breedSql = $dbConnection->prepare('SELECT `breed` FROM `breed`');
+$breedSql->execute();
+$breeds = $breedSql->fetchAll();
+
+for ($i = 0; $i < count($breeds); $i++) {
+    $breed = $breeds[$i]['breed'];
+    $id = $i + 1;
+    $dropdownBreeds .= "<option value=\"$id\">$breed</option>";
+}
+
+// If the user has selected a breed, get and show the cats
+if (isset($_GET['breed'])) {
+    $catHydrator = new TopCat\Hydrators\CatHydrator($dbConnection);
+    $cats = $catHydrator->createCatEntitiesArray((int)$_GET['breed']);
+    foreach ($cats as $cat) {
+        $catshtml .= '<div class="cat-image"><img src="' . $cat->getImage() . '" alt="A cat"></div>';
```

Comment:
Could you change the alt text to include the breed?

Reply:
on it

Reply:
Yes